Simplenote/Classes/SPEditorTextView.m

```diff
@@ -10,18 +10,21 @@
 #import "SPTagView.h"
 #import "SPInteractiveTextStorage.h"
 #import "NSMutableAttributedString+Styling.h"
-#import "NSString+Attributed.h"
-#import "UIDevice+Extensions.h"
 #import "VSTheme+Extensions.h"
 #import "Simplenote-Swift.h"
 
 NSString *const MarkdownUnchecked = @"- [ ]";
 NSString *const MarkdownChecked = @"- [x]";
 NSString *const TextAttachmentCharacterCode = @"\U0000fffc"; // Represents the glyph of an NSTextAttachment
 
+CGFloat const TagViewHeight = 44;
+CGFloat const TextViewTopInsets = 8;
+CGFloat const TextViewBottomInsets = TagViewHeight * 2;
```

Comment:
We probably want to get rid of having a fixed height as it doesn't play well with dynamic type.
Maybe add a TODO?

Reply:
Good point! At the moment the Tags Editor isn't Dynamic Type friendly, let's add a TODO.

Thank you!